tests/acceptance/testall

```diff
@@ -100,6 +100,7 @@ CF_PROMISES=${CF_PROMISES:-}        ; export CF_PROMISES
 CF_SERVERD=${CF_SERVERD:-}          ; export CF_SERVERD
 CF_EXECD=${CF_EXECD:-}              ; export CF_EXECD
 CF_KEY=${CF_KEY:-}                  ; export CF_KEY
+CF_KEYCRYPT=${CF_KEYCRYPT:-}        ; export CF_KEYCRYPT
```

Comment:
Here and multiple places below, CF_KEYCRYPT is still mentioned. Can we change all of them to cf-secret? Might be the same in master?

Reply:
Yeah. I noticed that too.

Reply:
Done. Added a commit here and the *master* PR is here: https://github.com/cfengine/core/pull/4205